demo/assets/materials/debug.material

```diff
@@ -1,11 +1,11 @@
 (
     passes: [
         (
-            phase: Some("Opaque"),
+            phase: Some("Wireframe"),
             fixed_function_state: (
                 alpha_blending: Enabled,
                 depth_testing: EnabledReverseZ,
-                polygon_mode: Some(Line)
+                fill_mode: Some(Wireframe)
```

Comment:
I wonder why this never threw an error before

Reply:
I thought it's because the struct is defined as `Option` so it was just defaulting to `None`. Unless you thought that unexpected fields would cause the parser to fail?

Reply:
yeah, ideally they would. they're almost always a mistake

Reply:
🤷